lua/mapvote/server/modules/rtv.lua

```diff
@@ -110,14 +104,14 @@ function RTV.CanVote( ply )
     end
 
     if ply.RTVVoted then
```

Comment:
NITPICK: might wanna store this as MapVote_RTVVoted or something like it, keeps it more "clean"